draft-ietf-httpbis-variants.md

```diff
@@ -676,7 +676,10 @@ Variant-Key: (gold europe)
 Vary: Cookie
 ~~~
 
+## Client Hints {#client-hints}
 
+Client Hints {{CLIENT-HINTS}} might be combined with Variants response header field to enable fine-grained control of the cache key for improved cache efficiency.
```

Comment:
The {{CLIENT-HINTS}} reference fails to build for me locally, but I'm not sure why...